js-miniapp-sdk/src/miniapp.ts

```diff
@@ -146,7 +148,22 @@ export class MiniApp implements MiniAppFeatures, Ad, Platform {
   }
 
   requestLocationPermission(): Promise<string> {
-    return this.requestPermission(DevicePermission.LOCATION);
+    const locationPermission = [
+      {
+        name: CustomPermissionName.LOCATION,
+        description: 'We would like to display the location of your device.',
```

Comment:
I think we should allow the developer to optionally pass the description, so maybe we could change this method to `requestLocationPermission(description?: string)`.

Also, I don't think we should assume what the purpose of requesting the location is, so could we make the description blank by default?

Reply:
Thanks Cory, I'll update this part as well.